frontend/src/main/components/MenuItems/MenuItemsTable.js

```diff
@@ -0,0 +1,76 @@
+import OurTable, { ButtonColumn } from "main/components/OurTable";
+import { useBackendMutation } from "main/utils/useBackend";
+import { onDeleteSuccess } from "main/utils/UCSBDateUtils"
```

Comment:
I don't think using UCSBDateUtils is a good idea, although onDeleteSuccess is the same function.